content/en/cloud-computing.md

```diff
@@ -8,7 +8,24 @@ tags: ["infrastructure", "fundamental", ""]
 ## What it is
 
 Cloud computing offers compute resources like CPU, network, and disk capabilities on-demand over the internet, allowing users to access and use computing power in a remote physical location. 
-We generally differentiate between private and public cloud, depending on whether the cloud infrastructure is exclusively dedicated to an organization or shared for open public services.
+It extends beyond merely providing resources and we generally differentiate between private and public cloud. 
```

Comment:
I wonder if we should expand the cloud computing definition to include the different service models of cloud computing too - IaaS (Infrastructure as a Service), PaaS (Platform as a Service), and SaaS (Software as a Service).

This may be out of scope for this definition tho. @iamNoah1, thoughts?

Reply:
Hmm...this section is already very long, and we want to be sure to stick to our [minimal viable definition](https://glossary.cncf.io/style-guide/#minimum-viable-definition) goal. We had separate definitions or issues to create them for IaaS, PaaS, and SaaS but decided not to include them because they were not cloud native specific or necessary to understand cloud native concepts. 

Going back to the minimal viable definition, what are the most important things readers need to understand about this concept to make sense of it when reading about it in other resources? And let's stick to that.

Reply:
I personally like the idea of at least mentioning that when we talk about cloud computing we have different abstractions like Paas, IaaS, SaaS, and even FaaS if you want. On the other side and correct me if I am wrong @CathPag @seokho-son @jihoon-seo at some point we decided to want to include XaaS, isn't it?! 
So maybe as a suggestion, let's say no to XaaS for this PR and now. Let's talk about this topic in the next glossary meeting, and then adjust accordingly.

Reply:
So, I will wait till the maintainers and the core team decides to include PaaS, IaaS, SaaS and FaaS. Drop a text here for confirmation after the meeting! @iamNoah1 @CathPag

Reply:
We will skip it for now, so closing this comment.